test/contlinear.jl

```diff
@@ -791,10 +791,15 @@ function linear8test(solver::MOI.AbstractSolver; atol=Base.rtoldefault(Float64),
             @test MOI.getattribute(m, MOI.TerminationStatus()) == MOI.Success
             @test MOI.getattribute(m, MOI.DualStatus()) == MOI.InfeasibilityCertificate
             @test MOI.cangetattribute(m, MOI.ConstraintDual(), c)
-            @test MOI.getattribute(m, MOI.ConstraintDual(), c) ≈ -1 atol=atol rtol=rtol
+            cd = MOI.getattribute(m, MOI.ConstraintDual(), c)
+            @test cd < -atol
             # TODO: farkas dual on bounds
```

Comment:
Maybe replace this comment with a link to https://github.com/JuliaOpt/MathOptInterface.jl/pull/127

Reply:
good idea